client/request_test.go

```
@@ -364,6 +433,39 @@ func Test_Request_Cookie(t *testing.T) {
+func Benchmark_Request_Cookies(b *testing.B) {
+	req := AcquireRequest()
+	req.SetCookies(map[string]string{
+		"foo": "bar",
+		"bar": "foo",
+	})
+
+	b.ResetTimer()
+	b.ReportAllocs()
+
+	for i := 0; i < b.N; i++ {
+		for range req.Cookies() {
```

golangci-lint (1.62.2)

[warning] empty-block: this block is empty, you can remove it

(revive)